pkg/virtctl/vmexport/vmexport.go

```diff
@@ -345,16 +360,22 @@ func (c *command) parseExportArguments(args []string, vmeInfo *VMExportInfo) err
 
 func (c *command) initVMExportInfo(vmeInfo *VMExportInfo) error {
 	vmeInfo.ExportSource = getExportSource()
-	vmeInfo.OutputFile = outputFile
 	// User wants the output in a file, create
 	if outputFile != "" && outputFile != "-" {
+		vmeInfo.OutputFile = outputFile
 		output, err := os.Create(vmeInfo.OutputFile)
 		if err != nil {
 			return err
 		}
 		vmeInfo.OutputWriter = output
 	} else {
 		vmeInfo.OutputWriter = c.cmd.OutOrStdout()
+		vmeInfo.OutputFile = ""
+		// Setting standard printer to output into stderr. We can then output
+		// the volume into stdout without any interfering prints.
+		setPrintToOutput(func(format string, a ...interface{}) (int, error) {
+			return fmt.Fprintf(os.Stderr, format, a...)
```

Comment:
I do wonder if we should always write to stderr but don't want to break any existing scripts/programs

Reply:
I think the name printToOutput is a bit confusing at first I thought you print to the same output as defined in the outputfile flag which is not the case, its either to stdout or stderr..
maybe customPrint or some short of sPrint, cPrint

Reply:
> printToOutput is a bit confusing at first I thought you print to the same output as defined in the output file flag

Didn't make that connection but you could be right. vmexport PRs are common so I can consider changing it in the near future.